data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Deep: Syndicate Convoy"
+	minor
+	name `Pickup from Hephaestus`
+	description `Escort a convoy of freighters to <stopovers> where they will pick up materials needed for scientific experiments. Return the convoy to <destination> by <date> for payment (<payment>).`
+	source
+		attributes "deep"
+		near "Gomeisa" 2
+	stopover "Hephaestus"
+	deadline
+	to offer
+		"combat rating" > 65
+		not "Deep: Mystery Cubes 4: done"
+		or
+			random < 20
+			and
+				"deep convoy" == 1
+				random < 30
+			and
+				"deep convoy" == 2
+				random < 40
+        
+	on offer
+		conversation
+			`You are approached by a man in a Deep Security uniform who asks if you would be willing to escort a few freighters to <stopovers> and back by <day>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
+			choice
+				`	"Sure, I can do that."`
+				`	"Sorry, but I won't be traveling in that direction any time soon."`
+					decline
+			
+			`	"Good. Before you leave, know that there may be pirate forces waiting for you in Alphard, Sirius, and Altair. You can try to avoid them, but make sure you get the freighters back on time."`
+			`	The Deep Security officer wishes you good luck and walks off, leaving you wonder how he knows where pirate fleets might be waiting for you.`
+				accept
+	
+	npc accompany save
+		government "Merchant"
+		personality timid escort
+		fleet
+			names "civilian"
+			variant
+				"Bulk Freighter" 2
+				"Mule"
+	npc
+		government "Merchant"
+		personality timid
+		fleet
+			fighters "civilian"
+			variant
+				"Dagger"
+			variant
+				"Boxwing"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Alphard"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Sirius"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system "Altair"
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis
+		system "Castor"
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis
+		system "Porrima"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system destination
+		fleet "Large Northern Pirates"
+	
+	on stopover
+		dialog `When you arrive at the pickup location it is already night on this side of <planet>. The Syndicate employees who load the freighters do not allow you to see what is inside the cargo, and advise you to leave quickly once the cargo is loaded.`
```

Comment:
Recommend changing the first sentence to "You arrive at the pickup location on the night side of <planet>." It makes the focus of the sentence "you" instead of the less specific "it" (referring to the time of day). This communicates the same information but makes the player sound more involved in the event described.